### PR TITLE
dependency updates

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -409,11 +412,48 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.exasol</groupId>
+                <artifactId>error-code-crawler-maven-plugin</artifactId>
+                <version>0.1.1</version>
```
Comment (jakobbraun) [inline pom.xml:418]: update to 5.0.0.

Comment (jakobbraun) [reply pom.xml:418]: for example using the mavenDependencyUpdate script

File: release_config.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+release-platforms:
+  - GitHub
+  - Community
```
Comment (jakobbraun) [inline release_config.yml]:
```suggestion
```

File: release_config.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  - Maven
+  - Jira
+community-tags:
+  - Row Level Security
+  - Security
+community-project-name: Row Level Security
+community-project-description: |
+  This projecr allows you to protect access to database tables on a per-row level based on roles and / or tenants.
```
Comment (jakobbraun) [inline release_config.yml]:
```suggestion
```

File: src/assembly/all-dependencies.xml [hunk 4]
```diff
@@ -14,9 +14,21 @@
     </containerDescriptorHandlers>
     <dependencySets>
         <dependencySet>
+            <outputDirectory>/</outputDirectory>
```
Comment (jakobbraun) [inline src/assembly/all-dependencies.xml:17]: Explain why

File: pom.xml (hunk 1 above)
Comment (AnastasiiaSergienko) [reply pom.xml:418]: Created a ticket to add error codes: https://github.com/exasol/row-level-security/issues/103

**Pull request merged**